### PR TITLE
NTR: owns; is owned by

### LINKED ISSUE #674: NTR: owns; is owned by
NTR 1: 
Label: owns
Elucidation: x owns y if and only if y is at x’s full disposal. This primitive relation is the foundation of the owner’s right to have the owned entity at his/her full disposal.  
Definition editor: Mathias Brochhausen

NTR 2:
Label: is owned by
Definition: The inverse of owns. 
Definition editor: Mathias Brochhausen

### PR DESCRIPTION
**Pull request by wdduncan**
Fixes #674

### CODE REVIEW

**Comment by wdduncan** [comment]
@mbrochhausen @CDowland  @zhengj2007 I've add the terms.  
Note: RO doesn't have an `elucidation` AP. So, I took some liberties with the definition.

Who do you suggest to review?

**Comment by zhengj2007** [comment]
I recommended @mbrochhausen.

**Comment by zhengj2007** [comment]
@wdduncan Could you also use the inverse of axiom to indicate that owns is inverse relation of 'is owned by' and vice versa? Besides, here is the definition of OBIB:0000735 [is owned by](http://purl.obolibrary.org/obo/OBIB_0000735) in OBIB:
A is owned by b if b has complete power over a. All rights and obligations of ownership are grounded in this (primitive) relation. The claims and obligations of ownership can be partially transferred to a third party by the owner, b.
Can we also add it in the RO and indicate the definition coming from OBIB:0000735 use 'definition source' ? Thanks!

**Comment by wdduncan** [comment]
@zhengj2007 I added the `inverse of` axiom for `owns` and `is owned by`, and the definition source (`dcterms:source`) for `is owned by` as OBIB:0000735.

As for the definition of `is owned by`, the wording is different. It uses the phrase "complete power", where as the definition for `owns` uses the phrase "full disposal". These seem subtly different to me.

Also, "complete power" seems a bit strong. I own my dog, but I don't have complete power over him.

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6791,6 +6822,11 @@ binary, and takes no temporal argument.
           develops_from(x,y)")
 AnnotationAssertion(rdfs:label obo:RO_0001902 "relation has no temporal argument"@en)
 
+# Individual: <https://orcid.org/0000-0001-9625-1899> (Bill Duncan)
```
Comment (cthoyt) [inline src/ontology/ro-edit.owl:6825]: i'm surprised this wasn't already in there, @matentzn is this how it should look for the ORCIDIO import?

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -6477,6 +6480,34 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017001 "2021-11-08T12:00:00Z"
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:RO_0017001 "utilizes")
 AnnotationAssertion(rdfs:label obo:RO_0017001 "device utilizes material"@en)
 
+# Object Property: obo:RO_0017002 (owns)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017002 "biobank organization equivalentClass:
```
Comment (cmungall) [inline src/ontology/ro-edit.owl:6485]: Keep example of usage to be simple

Comment (wdduncan) [reply src/ontology/ro-edit.owl]: I removed this example of usage.

File: src/ontology/ro-edit.owl (hunk 1 above)
Comment (wdduncan) [reply src/ontology/ro-edit.owl:6821]: Is this a blocker for approval of the PR?

Comment (cthoyt) [reply src/ontology/ro-edit.owl:6821]: No, sorry for the late reply (i had some personal things to deal with).

**Comment by wdduncan** [comment]
I think this is ready. @CDowland What do you think?

File: src/ontology/ro-edit.owl [hunk 3]
```diff
@@ -6542,6 +6545,30 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017001 "2021-11-08T12:00:00Z"
...
+AnnotationAssertion(rdfs:label obo:RO_0017008 "owns"@en)
+InverseObjectProperties(obo:RO_0017008 obo:RO_0017009)
+
+# Object Property: obo:RO_0017009 (is owned by)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017009 "freezer is owned by a biobank organization")
+AnnotationAssertion(Annotation(dc:source "OBIB:0000735") obo:IAO_0000115 obo:RO_0017009 "Inverse of the owns relation.")
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0001-9625-1899>)
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0003-1834-3856>)
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0003-1909-9269>)
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:6567]: Need to change the annotation to `terms:contributor`

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl (hunk 3 above)
Comment (CDowland) [reply src/ontology/ro-edit.owl:6567]: @wdduncan Yes, looks good, thanks!

Comment (wdduncan) [reply src/ontology/ro-edit.owl:6567]: @CDowland please see @anitacaron comment. Can you fix?

> Need to change the annotation to terms:contributor

File: src/ontology/ro-edit.owl (hunk 1 above)
Comment (anitacaron) [reply src/ontology/ro-edit.owl:6999]: I'm checking this now; it should be in the orcidio_import.owl, not the edit file. Have you added this manually, @wdduncan?

Comment (anitacaron) [reply src/ontology/ro-edit.owl:6999]: You should request to add your information in orcidio https://github.com/cthoyt/orcidio. Then, when available, we can update the orcidio_import. Make sure to remove the individual from the edit file, please.

Comment (wdduncan) [reply src/ontology/ro-edit.owl]: Ok. This has been done.

Comment (anitacaron) [reply src/ontology/ro-edit.owl]: You must add the individual to the repository https://github.com/cthoyt/orcidio, not in the `orcidio_import.owl` here. This change will be lost when update the `orcidio_import.owl`

Comment (anitacaron) [reply src/ontology/ro-edit.owl]: `orcidio_import.owl` is like a normal import using ODK, the same as importing terms from OBI or GO. The `orcidio_import.owl` is not the same as the repository https://github.com/cthoyt/orcidio

Comment (wdduncan) [reply src/ontology/ro-edit.owl]: Thanks @anitacaron   
My question was regrading whether the @cthoyt `orcidio_import.owl` file was generated using a script or manually curated.

Comment (wdduncan) [reply src/ontology/ro-edit.owl]: @anitacaron I removed myself from `ordidio_import.owl`.  
I think I should be in the `orcidio_import.owl` file as "William D Duncan". I am in the [file](https://github.com/cthoyt/orcidio/blob/c670db35108e0f8c0716d858b0dcba2a2c928b8b/orcidio.owl#L2894) on https://github.com/cthoyt/orcidio.

**Comment by wdduncan** [comment]
@anitacaron I am in the oridio on Charley's github site. So, is there an import problem for RO? What needs to be done to update the RO ordicio?

**Comment by anitacaron** [comment]
@wdduncan, did you try to run `sh run.sh make imports/orcidio_import.owl`

File: src/ontology/ro-edit.owl [hunk 4]
```diff
@@ -2628,7 +2630,6 @@ SubObjectPropertyOf(obo:RO_0002237 obo:RO_0002444)
 # Object Property: obo:RO_0002238 (has component participant)
 
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0002238 "X 'has component participant' Y means X 'has participant' Y and there is a cardinality constraint that specifies the numbers of Ys.")
-AnnotationAssertion(obo:IAO_0000117 obo:RO_0002238 <https://orcid.org/0000-0001-9625-1899>)
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:2631]: @wdduncan, all your ORCID entries were removed in the edit file. This explains why your instance is no longer in the `imports/orcidio_import.owl`. It only imports information for all the ORCIDs present in the edit file. Please, add again using the annotation `term:contributor`.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl (hunk 4 above)
Comment (wdduncan) [reply src/ontology/ro-edit.owl:2631]: @anitacaron I think this checks out now. Please confirm.

Comment (anitacaron) [reply src/ontology/ro-edit.owl:2634]: You didn't add back all your contributions. Do you confirm you want to remove them? It's missing in RO:0002238 and RO:0017001.

Comment (wdduncan) [reply src/ontology/ro-edit.owl:2634]: Thanks @anitacaron! I missed that. I added back my contributions. I also changed the attribution to `https://orcid.org/0000-0003-2620-0345` (Asiyah) to `dcterms:contributor`.

Comment (anitacaron) [reply src/ontology/ro-edit.owl:2634]: Thanks @wdduncan!

**Review by anitacaron — APPROVED** [review]
Technically, it's approved. It still needs an ontology review.

**Comment by wdduncan** [comment]
Thanks @anitacaron!

@cmungall @cthoyt @CDowland can you please review.

**Comment by wdduncan** [comment]
@balhoff @CDowland can you please review this? It would be nice to get this merged before the next RO release.

**Review by CDowland — APPROVED** [review]

**Comment by CDowland** [comment]
@wdduncan I did a review/approval yesterday as requested, but it's still showing that another is required. It says "reviewers with write access," which I don't think includes me.

**Comment by wdduncan** [comment]
@cthoyt @balhoff @zhengj2007 @cmungall @matentzn @jamesaoverton @lschriml   
Can one of you please review this PR?

**Review by cthoyt — APPROVED** [review]
besides the spurious diffs, it looks fine

**Comment by zhengj2007** [comment]
@wdduncan Could you please update the dcterms:source for 'owns'? It should be OBIB:0000732.

Since I was not assigned as the reviewer, I cannot review and approve the request.

**Comment by wdduncan** [comment]
@zhengj2007 can you please approve this PR.

**Comment by wdduncan** [comment]
@zhengj2007 I can't add you as a reviewer. It seems that you are not a member of obo-rel repo ... sorry I thought you were.

@anitacaron Can I add her so that she can review this PR? If so, what level of permissions does Jie need?

**Comment by anitacaron** [comment]
@wdduncan I added @zhengj2007 with a 'write' role; we need to wait for her to accept the invite to add her as a reviewer here.

**Comment by wdduncan** [comment]
Thanks @anitacaron!

**Comment by zhengj2007** [comment]
> @wdduncan I added @zhengj2007 with a 'write' role; we need to wait for her to accept the invite to add her as a reviewer here.

@anitacaron Thanks for invitation. 
@wdduncan I have accepted the invitation. You can assign me to review the request now.

**Comment by wdduncan** [comment]
Thanks Jie. I assigned you as a reviewer.

**Review by zhengj2007 — CHANGES_REQUESTED** [review]
@wdduncan According to Mathias' comments: "Based on the OBIB discussion, we do not need the DCterms: source.", RO may remove "DCterms: source" annotations associated with 'owns' and 'is owned by'. If RO prefers to keep the DCterms: source annotation to reference OBIB terms, it need to change "DCterms: source OBIB:0000735" to "DCterms: source OBIB:0000732" to term RO:0017008 'owns'.

**Comment by wdduncan** [comment]
@zhengj2007 I believe @mbrochhausen said that the `dcterms:source` annotation was not needed for the `owns/is owned by`. Am I misunderstanding you @mbrochhausen .

AFAIK, `dcterms:source` is not required. If you think it is required can you please add it.

**Comment by zhengj2007** [comment]
> @zhengj2007 I believe @mbrochhausen said that the `dcterms:source` annotation was not needed for the `owns/is owned by`. Am I misunderstanding you @mbrochhausen .
> 
> AFAIK, `dcterms:source` is not required. If you think it is required can you please add it.

@wdduncan Yes. However, it is still in ro-edit.owl. So, it should be either removed or corrected if you want to keep it.

**Comment by wdduncan** [comment]
Ah ... okay. Sorry @zhengj2007 I misunderstood you :)

I removed the `dcterms:source` annotation for the definition of `is owned by`.
image

**Comment by wdduncan** [comment]
Thanks @zhengj2007!  
I removed it.

**Comment by zhengj2007** [comment]
Actually 4 reviewers approved this pull request. Don't know why it only shows 1 approval.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**